experimenter/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_ready_for_review_serializer.py

```diff
@@ -3237,6 +3089,435 @@ def test_validate_feature_versioned_localized(self):
+        [
+            (
+                NimbusExperiment.Version.FIREFOX_120,
+                NimbusExperiment.Version.NO_VERSION,
+                NimbusConstants.ERROR_FEATURE_CONFIG_UNSUPPORTED_IN_VERSION.format(
+                    feature_config="FEATURE",
+                    version="121.0.0",
+                ),
+            ),
+            (
+                NimbusExperiment.Version.FIREFOX_121,
+                NimbusExperiment.Version.FIREFOX_122,
+                NimbusConstants.ERROR_FEATURE_CONFIG_UNSUPPORTED_IN_RANGE.format(
+                    feature_config="FEATURE",
+                ),
+            ),
+        ]
+    )
+    def test_fml_validate_feature_versioned_unsupported_versions(
+        self,
+        min_version,
+        max_version,
+        expected_error,
+    ):
+        application = NimbusExperiment.Application.FENIX
+
+        feature = NimbusFeatureConfigFactory.create(
+            application=application,
+            slug="FEATURE",
+            name="FEATURE",
+            schemas=[
+                NimbusVersionedSchemaFactory.build(version=None, schema=None),
+                NimbusVersionedSchemaFactory.build(
+                    version=self.versions[(120, 0, 0)], schema=None
+                ),
+                NimbusVersionedSchemaFactory.build(
+                    version=self.versions[(122, 0, 0)], schema=None
+                ),
+            ],
+        )
+
+        NimbusFeatureConfigFactory.create(
+            application=application,
+            schemas=[
+                NimbusVersionedSchemaFactory.build(version=None, schema=None),
+                NimbusVersionedSchemaFactory.build(
+                    version=self.versions[(120, 0, 0)], schema=None
+                ),
+                NimbusVersionedSchemaFactory.build(
+                    version=self.versions[(121, 0, 0)], schema=None
+                ),
+                NimbusVersionedSchemaFactory.build(
+                    version=self.versions[(122, 0, 0)], schema=None
+                ),
+            ],
+        )
+
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.CREATED,
+            targeting_config_slug=NimbusExperiment.TargetingConfig.NO_TARGETING,
+            application=application,
+            firefox_min_version=min_version,
+            firefox_max_version=max_version,
+            feature_configs=[feature],
+        )
+
+        for branch in experiment.treatment_branches:
+            branch.delete()
+
+        serializer = NimbusReviewSerializer(
+            experiment,
+            data=NimbusReviewSerializer(experiment, context={"user": self.user}).data,
+            context={"user", self.user},
+        )
+
+        self.assertFalse(serializer.is_valid())
+        self.assertTrue(
+            serializer.errors["reference_branch"]["feature_values"][0]["value"][
+                0
+            ].startswith(expected_error),
```

Comment:
why `startswith`? Why not an equality compare?

Reply:
This is what is being used in most of the other comparisons for the serializer warnings and serializer errors. The error that it's `startswith`ing is the full error. I'm happy to change this to use `assertIn` like the others if you'd like?

Reply:
Sometimes when we use `assertIn` it'll look like this:
```
AssertionError: 'Feature Manifest errors occurred during validation' not found in ErrorDetail(string='Feature Manifest errors occurred during validation', code='invalid')
````